build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id 'archunit.base-conventions'
-    id 'com.github.johnrengelman.shadow' version '7.1.2' apply false
+    id 'com.gradleup.shadow' version '8.3.5' apply false
```

Comment:
After applying the `fixup!`, the first commit's message shouldn't be
> upgrade Gradle Shadow plugin to version `8.1.1`

anymore. 😉

Reply:
Huh, I thought I adjusted that 🤯 Weird...

Reply:
Are you sure you reviewed the latest state? When I check the 3 commits it says "upgrade Gradle Shadow plugin to version `8.3.5`"

Reply:
Ooh, sorry: I was indeed not... 🙈 That's a wonderful commit message!